src/Apps/Artwork/Components/OtherAuctions.tsx

```diff
@@ -1,61 +1,27 @@
-import { Button, Flex, Serif } from "@artsy/palette"
```

Comment:
The corresponding story for this can probably just be removed.

It would need an update b/c of the changes here. Currently, this component was 'dumb' in that, it just rendered the children in some sort of flex/list format. And the story included the similarly 'dumb' `AuctionCard` component as the children.

Now, `OtherAuctions` directly references the _fragment container_ `AuctionCard`. So the story, at a minimum needs to be updated with the new data/fixture, as well as a `<RelayStubProvider>` wrapper (or other options).

But, it can probably just be removed also?